src/Microsoft.Extensions.Configuration.Abstractions/IConfigurationSection.cs

```diff
@@ -22,5 +22,10 @@ public interface IConfigurationSection : IConfiguration
         /// Gets or sets the section value.
         /// </summary>
         string Value { get; set; }
+
+        /// <summary>
+        /// Determines whether the section has a <see cref="Value"/> or has children 
+        /// </summary>
+        bool Exists();
```

Comment:
You should do this as an extension method instead, modifying interfaces is a breaking change